openvidu-browser/src/OpenVidu/Stream.ts

```diff
@@ -217,6 +217,8 @@ export class Stream extends EventDispatcher {
         super();
 
         this.session = session;
+        this.audioActive = false;
+        this.videoActive = false;
```

Comment:
This initialization is unnecesary because of the boolean variables has false by default.
Please, remove it.

Reply:
@CSantosM I agree with you when it comes to TypeScript. But the same does not apply to the translated JavaScript. 
Below is the `Stream.js` available at `node_modules/openvidu-browser/lib/OpenVidu/Stream.js`:

```
var Stream = /** @class */ (function (_super) {
    __extends(Stream, _super);
    /**
     * @hidden
     */
    function Stream(session, options) {
        var _this = _super.call(this) || this;
        ...
        if (options.hasOwnProperty('id')) {
            // InboundStreamOptions: stream belongs to a Subscriber
            _this.inboundStreamOpts = options;
            _this.streamId = _this.inboundStreamOpts.id;
            _this.creationTime = _this.inboundStreamOpts.createdAt;
            _this.hasAudio = _this.inboundStreamOpts.hasAudio;
            _this.hasVideo = _this.inboundStreamOpts.hasVideo;
            if (_this.hasAudio) {
                _this.audioActive = _this.inboundStreamOpts.audioActive;
            }
            if (_this.hasVideo) {
                _this.videoActive = _this.inboundStreamOpts.videoActive;
                ...
            }
            ...
        }
```

Notice that `audioActive` and `videoActive` were never initialized. By making changes to the `Stream.js` and changing the other changes on this PR but not this, I ran into another issue that was the lack of these properties set on the exact same scenario described on the issue.

Reply:
@CSantosM Actually, as I'm remembering, the issue I faced is not that the browser threw an error about the variables being `undefined`. But OpenVidu server returned an error because the payload sent to the server by the `Session` class didn't have the `videoActive`. JavaScript was omitting it from the payload object on the line below when these are undefined.
https://github.com/OpenVidu/openvidu/blob/5d479dc645959f51ed7c56696d70e8e779f07d88/openvidu-browser/src/OpenVidu/Session.ts#L1211

Notice that the "happy path" contains a safe-check to address this:

https://github.com/OpenVidu/openvidu/blob/5d479dc645959f51ed7c56696d70e8e779f07d88/openvidu-browser/src/OpenVidu/Session.ts#L1185